cmd/server.go

```diff
@@ -275,6 +277,10 @@ var boolFlags = map[string]boolFlag{
 	},
 }
 var intFlags = map[string]intFlag{
+	ParallelPlansPoolSize: {
+		description:  "Max size of the wait group that runs parallel plans (if enabled).",
+		defaultValue: DefaultParallelPlansPoolSize,
```

Comment:
Do you find yourself tuning this number? Just wondering if we need to expose it.

Reply:
I didn't end up tuning it after it was set, probably not worth exposing 👍